### ISSUE TITLE
Embedding tweets breaks when the account goes private

### ISSUE THREAD

**Issue by richburroughs** [PaulieScanlon/mdx-embed#248]
**Describe the bug**
It looks like embedded tweets fail when the account goes private. We use Netlify and our build broke. 

I know the person who had made the tweet and she did go private for a bit right before we got the error.

We saw this in the logs:

ERROR 2022/10/31 16:38:49 Failed to get JSON resource "https://api.twitter.com/1/statuses/oembed.json?hide_thread=1&id=1334188913724850177": Failed to retrieve remote file: Forbidden, body: "{\"error\":\"Sorry, you are not authorized to see this status.\",\"request\":\"\\/1\\/statuses\\/oembed.json?hide_thread=1&id=1334188913724850177\"}"

**To Reproduce**
1. Embed a tweet
2. Set the Twitter account to private
3. See error

**Expected behavior**

Something less brittle. I think I'd probably expect it to just not display the tweet but log something.

On a related topic, I expect more people will be deleting accounts too with everything happening there at Twitter, too. Not sure how the code handles that.

**Comment by spences10** [comment]
Yeah, good point @richburroughs maybe something in the `twttrLoad`, `util.ts` method to gracefully close out the component!

https://github.com/PaulieScanlon/mdx-embed/blob/main/packages/mdx-embed/src/components/twitter/utils.ts#L7

feel free to contribute if you have another suggestion or discuss it further here.

@PaulieScanlon I presume you're of the same opinion on the suggested approach here?

**Comment by PaulieScanlon** [comment]
Hi @richburroughs could you provide the full `tweetLink` including the username so i can see what you were seeing.

I'm currently working on upgrading MDX Embed to use MDX 2 and will investigate this issue after i've completed the upgrade.

**Comment by PaulieScanlon** [comment]
@spences10 thanks babes 😘

**Comment by richburroughs** [comment]
The link wouldn't help as the person is no longer private. They had just gone private for a few days.

It should be easy to duplicate though by linking to a tweet from any other private account.

Thanks :)

**Comment by PaulieScanlon** [comment]
Thanks @richburroughs do you have a link to a Tweet from a private account to help me reproduce?

**Comment by richburroughs** [comment]
Hey Paulie,

Sure, here's one:

https://twitter.com/giflian/status/1605757037455216640
